src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -111,8 +111,9 @@ public synchronized void launch(SlaveComputer computer, TaskListener listener) {
         }
 
         String cloudName = node.getCloudName();
-        final PodTemplate template = node.getTemplate();
```

Comment:
https://github.com/jenkinsci/kubernetes-plugin/blob/ba6b8936497d51c8075bef4d9aa9940b33300813/src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java#L93-L95 IIUC. Could alternately use `getTemplateOrNull` and explicitly throw an error here.